test/server/cards/01_SOR/events/OpenFire.spec.js

```diff
@@ -24,6 +24,8 @@ describe('Open Fire', function() {
 
                 this.player1.clickCard(this.cartelSpacer);
                 expect(this.cartelSpacer).toHaveExactUpgradeNames([]);
```

Comment:
Thanks for fixing all this - I also realized that this line is maybe more clear as `expect(this.cartelSpacer.isUpgraded()).toBe(false)`